src/main/java/com/github/webicitybrowser/spec/fetch/imp/FetchEngineImp.java

```diff
@@ -1,14 +1,16 @@
 package com.github.webicitybrowser.spec.fetch.imp;
 
-import com.github.webicitybrowser.spec.fetch.FetchEngine;
-import com.github.webicitybrowser.spec.fetch.FetchParameters;
-import com.github.webicitybrowser.spec.fetch.FetchParams;
-import com.github.webicitybrowser.spec.fetch.FetchResponse;
+import com.github.webicitybrowser.spec.fetch.*;
```

Comment:
As mentioned earlier, we are not using wildcard imports
You can use your IDE's auto-import or organize imports feature

Reply:
I have to watch this more carefully because my IDE is adding this wildcard automatically

Reply:
Strange. What IDE are you using?